sec.club/src/App.js

```diff
@@ -85,6 +86,7 @@ function App() {
             </Switch>
           </Suspense>
         </Suspense>
+        <Footer/>
```

Comment:
This needs to go inside the suspense tag
```suggestion
        <Footer/>
        </Suspense>
```